### PR TITLE
Refactor/touchstone

### LINKED ISSUE #54: Use touchstone library instead of themis/xmetrics
As an addition to this switch, use `testutil` and `expfmt` to verify prometheus metrics

### PR DESCRIPTION
**Pull request by j-mai**
Use `touchstone` instead of `themis/xmetrics`

Closes #54

### CODE REVIEW

**Review by kristinapathak — COMMENTED** [review]
Is this tested? Do you still get bascule capability metrics?

File: deploy/docker-compose/docker-compose.yml
Comment (j-mai) [reply deploy/docker-compose/docker-compose.yml:62]: yeah the dockerfile for caduceator is incorrect-pushing a fix for that!

File: main.go
Comment (kristinapathak) [inline main.go:134]: This might make sense as a `Provide()` function in wrp-listener? What do you think?

Comment (j-mai) [reply main.go:134]: yeah that makes sense! I'll add it to wrp-listener.

**Review by kristinapathak — APPROVED** [review]
This looks great! I would suggest waiting to merge this until after merging+releasing the pr in wrp-listener (so the go.mod doesn't need a replace).

**Pull request merged**
